package.json

```diff
@@ -1,51 +1,48 @@
 {
-  "name": "newspack-listings",
-  "version": "2.14.1",
-  "description": "",
-  "scripts": {
-    "cm": "newspack-scripts commit",
-    "semantic-release": "newspack-scripts release --files=newspack-listings.php",
-    "start": "npm ci && newspack-scripts watch",
-    "watch": "newspack-scripts watch",
-    "build": "newspack-scripts build",
-    "test": "echo \"Error: no test specified\" && exit 0",
-    "clean": "rm -rf dist/",
-    "lint": "npm run lint:scss && npm run lint:js",
-    "lint:js": "eslint --ext .js,.jsx src",
-    "lint:js:staged": "eslint --ext .js,.jsx",
-    "lint:php:staged": "./vendor/bin/phpcs",
-    "lint:scss": "stylelint '**/*.scss' --customSyntax postcss-scss --config=./node_modules/newspack-scripts/config/stylelint.config.js",
-    "lint:scss:staged": "stylelint --customSyntax postcss-scss --config=./node_modules/newspack-scripts/config/stylelint.config.js",
-    "format:js": "prettier 'src/**/*.{js,jsx}' --write",
-    "format:scss": "prettier --write 'src/**/*.scss'",
-    "format:php": "./vendor/bin/phpcbf .",
-    "release:archive": "rm -rf release && mkdir -p release && rsync -r . ./release/newspack-listings --exclude-from='./.distignore' && cd release && zip -r newspack-listings.zip newspack-listings",
-    "release": "npm run build && npm run semantic-release"
-  },
-  "lint-staged": {
-    "*.js": "npm run lint:js:staged",
-    "*.scss": "npm run lint:scss:staged",
-    "*.php": "npm run lint:php:staged"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Automattic/newspack-listings.git"
-  },
-  "license": "GPL-2.0-or-later",
-  "bugs": {
-    "url": "https://github.com/Automattic/newspack-listings/issues"
-  },
-  "dependencies": {
-    "newspack-components": "^2.2.1"
-  },
-  "devDependencies": {
-    "@rushstack/eslint-patch": "^1.7.2",
-    "eslint": "^7.32.0",
-    "lint-staged": "^15.2.0",
-    "newspack-scripts": "^5.2.1",
-    "postcss": "^8.4.33",
-    "postcss-scss": "^4.0.9",
-    "prettier": "npm:wp-prettier@^2.6.2-beta-1",
-    "stylelint": "^15.11.0"
-  }
-}
+	"name": "newspack-listings",
+	"version": "2.15.0",
+	"description": "",
+	"scripts": {
+		"cm": "newspack-scripts commit",
+		"semantic-release": "newspack-scripts release --files=newspack-listings.php",
+		"build": "newspack-scripts wp-scripts build",
+		"start": "npm ci && npm run watch",
+		"watch": "newspack-scripts wp-scripts start",
+		"test": "echo 'No JS unit tests in this repository.'",
+		"lint": "npm run lint:scss && npm run lint:js",
```

Comment:
Running into this error when running this:

![Screenshot 2024-06-21 at 12 05 47](https://github.com/Automattic/newspack-listings/assets/17905991/19f73500-464c-4a8a-8d87-17f9f31e8242)

Likely related to this comment in slack: p1718927940733569/1718662684.817909-slack-C015W6BES8J

Reply:
The stylelint errors should be fixed as of `newspack-scripts@5.6.0-alpha.7`